arrows/mvg/applets/init_cameras_landmarks.h

```diff
@@ -0,0 +1,62 @@
+/*ckwg +29
+ * Copyright 2020 by Kitware, Inc.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *  * Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ *  * Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ *  * Neither name of Kitware, Inc. nor the names of any contributors may be used
+ *    to endorse or promote products derived from this software without specific
+ *    prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE AUTHORS OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#ifndef KWIVER_ARROWS_MVG_APPLETS_INIT_CAMERAS_LANDMARKS_H
+#define KWIVER_ARROWS_MVG_APPLETS_INIT_CAMERAS_LANDMARKS_H
+
+#include <vital/applets/kwiver_applet.h>
+#include <arrows/mvg/applets/kwiver_algo_mvg_applets_export.h>
+
+namespace kwiver {
+namespace arrows {
+namespace mvg {
+
+class KWIVER_ALGO_MVG_APPLETS_EXPORT init_cameras_landmarks
+  : public kwiver::tools::kwiver_applet
+{
+public:
+  init_cameras_landmarks();
+
+  PLUGIN_INFO( "init_cameras_landmarks",
```

Comment:
Not something to worry about now, but we may ultimately want to change the name of this tool, and probably change the name of the algorithm it calls.  It's not longer about "initializing" as it was in the early days of MAP-Tk.  This is the full estimation process.  So "estimate_cameras_landmarks" is more correct.  However, we might just call it "sfm" for structure-from-motion because that is the more commonly used term for this.

No need to change anything now, I just wanted to put that out for discussion.

Reply:
For now we should change underscores to hyphens in this name as that is our convention for applet names.  So `init-cameras-landmarks`